src/snapred/backend/dao/ingredients/ReductionIngredients.py

```diff
@@ -12,7 +12,7 @@ class ReductionIngredients(BaseModel):
 
     runConfig: RunConfig
     reductionState: ReductionState
-    pixelGroupingParameters: List[PixelGroupingParameters]
+    pixelGroupingParameters: List[PixelGroupingParameters] = Field(min_items=1)
```

Comment:
when this branch gets merged with next this will probably have to be replaced by PixelGroup.